packages/@dcl/ecs/src/systems/crdt/index.ts

```diff
@@ -151,6 +152,9 @@ export function crdtSceneSystem(engine: PreEngine, onProcessEntityComponentChang
 
             onProcessEntityComponentChange && onProcessEntityComponentChange(msg.entityId, msg.type, component, value)
           }
+        } else {
+          // TODO: test this line, it is fundammental to make the editor work
```

Comment:
check this

Reply:
Yes, I saw this but I wouldn't know how to test this properly, or even why is important for the editor. Will ask @menduz for help here and add a test in a new PR 👍